smartcontractkit/external-adapters-js
# [Fix] Metalsapi includes symbols for batched data

## Description
Fix to `metalsapi` missing sending `symbols` to the data provider.

NOTE: doesn't need a new changeset as @austinborn 's last PR added it for this

## Changes
- Uses inputParameters type to stop `base` as an array
- Sends `symbols` with data provider request
- Amend tests



## Steps to Test
1. Run `metalsapi` adapter and redux dev tools
2. Request a pair
3. The pair should be seen in redux dev tools subscription state, along with starting a batch warmer for that pair
4. Request another pair with the same `base`. It should join the previously made batch warmer.
5. Request a pair with a different `base`. It should start a new batch warmer because base is not a batchable property.
6. Request `base` as an array. It should fail with 400.

## Quality Assurance
